coordinator/src/main/java/com/tencent/rss/coordinator/AccessManager.java

```diff
@@ -36,11 +37,14 @@
 
   private final CoordinatorConf coordinatorConf;
   private final ClusterManager clusterManager;
+  private final Configuration hadoopConf;
```

Comment:
why Configuration should be added as a member?

Reply:
The coordinator should use a single hadoop conf, which should be init when the coordinator server starts, and the checker get it from uppper layer (AccessManger).